reporter/src/main/kotlin/reporters/evaluatedmodel/EvaluatedModelMapper.kt

```diff
@@ -445,13 +455,27 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
                 issues += addIssues(this.issues, EvaluatedOrtIssueType.ANALYZER, dependency, null, packagePath)
             }
 
+            val children = visitDependencies { dependencies ->
+                dependencies.map { node ->
+                    if (!deduplicateDependencyTree || node.getInternalId() !in visitedNodes) {
+                        val childNode = node.toEvaluatedTreeNode(scope, path + dependency)
+                        visitedNodes += node.getInternalId() to childNode
+                        childNode
+                    } else {
+                        // If this node has already been visited, cut the subtree here. So only return the node
```

Comment:
I'm not sure how much this is just personal preference: to me it seems that swapping the code of ìf`and `else` blocks, inverting the 'if´-condition would be easier to read. IMO this would also allow dropping the comment "If this node has already been visited" as it's obvious from the condition then.

Reply:
Reverted the condition, but I decided to keep a part of the comment.